policy/datasets.yaml

```diff
@@ -1398,17 +1398,10 @@ measurePhotonTransferCurve_metadata:
     storage: YamlStorage
     python: lsst.daf.base.PropertySet
     template: ''
-measurePhotonTransferCurveGainAndNoise:
-    description: "Persisted output data (gain and noise) from the MeasurePhotonTransferCurveTask"
-    persistable: ignored
-    python: lsst.cp.pipe.ptc.PtcDataset
-    storage: PickleStorage
-    tables: 'raw'
```

Comment:
I noticed that you left out "tables: raw" in the new definition. However, Robert had to add this again to fix a runtime error (see DM-22227).

Reply:
This is a good point, I'll check this out.

Reply:
Excellent catch, this totally needed fixing to not revert those changes, thank you!